src/tox_gh/plugin.py

```diff
@@ -80,14 +80,37 @@ def tox_add_core_config(core_conf: ConfigSet, state: State) -> None:
         state.conf.core.loaders.insert(0, MemoryLoader(env_list=env_list))
 
 
+installing = False
+
+
+@impl
+def tox_on_install(tox_env: ToxEnv, arguments: Any, section: str, of_type: str) -> None:  # noqa: ANN401, ARG001
+    """
+    Run before installing to prepare an environment.
+
+    :param tox_env: the tox environment
+    :param arguments: installation arguments
+    :param section: section of the installation
+    :param of_type: type of the installation
+    """
+    global installing  # noqa: PLW0603
```

Comment:
We should store state in the environment via tox_env https://tox.wiki/en/4.6.4/plugins_api.html#tox.config.sets.ConfigSet.add_constant

Reply:
I saw `add_constant`, but this isn't a constant, it's value changes.  Maybe I don't understand what constant means here?  Can you point me to an example that stores state in the way you like? I tried searching GitHub.

Reply:
It is a constant no? You set it once? A constant can have two states unset and set. (it's a different conversation what its value is).

Reply:
It flips between true (line 98) and false (line 112). It's not configuration, so it seems odd to have to pick a key and a description for it.  The description for `add_constant` says, "Add a constant value."

Reply:
Sounds to me like `add_constant("install_triggered", True)`